packages/expo-notifications/src/getExpoPushTokenAsync.ts

```diff
@@ -52,7 +52,12 @@ export default async function getExpoPushTokenAsync(
   const devicePushToken = options.devicePushToken || (await getDevicePushTokenAsync());
 
   const deviceId = options.deviceId || (await getDeviceIdAsync());
-  const projectId = options.projectId || Constants.easConfig?.projectId;
+  // Depending on the runtime environment, the default may be located in various places.
+  // @see https://github.com/expo/expo/issues/23225
```

Comment:
No need for PR link in comment (can navigate via blame if needed)
```suggestion
  // Depending on the runtime environment, the default may be located in various places.
```

Reply:
This is the format that's in all the other entries and is suggested by the GitHub bot (so that may need to be updated...?)

Reply:
@bradjones1 I believe you confused the review comment as being related to changelog. It's not - it's a comment for code. Can you please accept and rebuild? 
Thank you